src/Modules/CrestApps.OrchardCore.AI.Chat/Controllers/AdminController.cs

```diff
@@ -200,6 +208,75 @@ public async Task<ActionResult> HistoryPost(string profileId)
     public IActionResult Chat(string profileId)
         => RedirectToAction(nameof(Index), new { profileId });
 
+    [Admin("ai/chat/session/delete/{sessionId}", "DeleteChatSession")]
+    public async Task<IActionResult> Delete(string sessionId, string profileId)
```

Comment:
Why do we need string profileId paramter here? If we already have the session, it'll tell us the profileId.

Reply:
Also, add `[Authorize]` to only allow logged users to use this function